src/javascript/app_2/App/Containers/SettingsDialog/settings-chart.jsx

```diff
@@ -1,19 +1,32 @@
-import PropTypes       from 'prop-types';
-import React           from 'react';
-import { localize }    from '_common/localize';
-import { connect }     from 'Stores/connect';
-import Localize        from 'App/Components/Elements/localize.jsx';
+import PropTypes                     from 'prop-types';
+import React                         from 'react';
+import { localize }                  from '_common/localize';
+import { connect }                   from 'Stores/connect';
+import Localize                      from 'App/Components/Elements/localize.jsx';
 import MediaItem, {
     MediaHeading,
     MediaIcon,
     MediaDescription,
-}                      from 'App/Components/Elements/Media';
-import Checkbox        from 'App/Components/Form/Checkbox';
-import RadioGroup      from 'App/Components/Form/Radio';
+}                                    from 'App/Components/Elements/Media';
+import Checkbox                      from 'App/Components/Form/Checkbox';
+import RadioGroup                    from 'App/Components/Form/Radio';
+import OHLCDisabledLight             from 'Images/app_2/settings/OHLC-disabled.svg';
+import OHLCEnabledLight              from 'Images/app_2/settings/OHLC-enabled.svg';
+import ChartPositionEnabledLight     from 'Images/app_2/settings/bottom.svg';
+import OHLCDisabledDark              from 'Images/app_2/settings/dark/OHLC-disabled.svg';
+import OHLCEnabledDark               from 'Images/app_2/settings/dark/OHLC-enabled.svg';
+import ChartPositionEnabledDark      from 'Images/app_2/settings/dark/bottom.svg';
+import IntervalDurationDisabledDark  from 'Images/app_2/settings/dark/interval-disabled.svg';
+import IntervalDurationEnabledDark   from 'Images/app_2/settings/dark/interval-enabled.svg';
+import ChartPositionDisabledDark     from 'Images/app_2/settings/dark/left.svg';
+import IntervalDurationDisabledLight from 'Images/app_2/settings/interval-disabled.svg';
+import IntervalDurationEnabledLight  from 'Images/app_2/settings/interval-enabled.svg';
+import ChartPositionDisabledLight    from 'Images/app_2/settings/left.svg';
```

Comment:
Would it be worth naming these as `...Icon`? e.g. ChartPositionDisabledDarkIcon

Reply:
Yeap. Fine by me. Will do.